docs/source/implementations/labeling_excess_median.rst

```diff
@@ -0,0 +1,87 @@
+.. _implementations-labeling_excess_median:
+
+============================
+Labelling Excess Over Median
+============================
+
+Labelling according to excess over median is a binary labelling method used in the following paper `Zhu, M., Philpotts, F. and
+Stevenson, M., 2012. The benefits of tree-based models for stock selection.
+Journal of Asset Management, 13(6), pp.437-448. <https://link.springer.com/article/10.1057/jam.2012.17>`_
+
+In this method, a cross-sectional dataset of close prices of many different tickers is used. The median return at each time index
+is calculated and used as a proxy for market return. The median return is then subtracted from each observation's return to find the
+numerical excess return over median. If desired, the numerical values can be converted to categorical values according to the sign of
+the excess return. The labels can then be used in training regression and decision tree models.
+
+At time :math:`t`:
+
+.. math::
+
+    P_t &= \{p_{t,0}, p_{t,1}, ... p_{t,n}\}
+
```

Comment:
There must be a better way to space these formulas. @PanPip please will you help with this.

Reply:
@Jackal08 Will do

Reply:
Let's use this:
<pre>
.. math::
    :nowrap:

    \begin{gather*}
    P_t = \{p_{t,0}, p_{t,1}, ... p_{t,n}\} \\

    m = median(P_t) \\

    L(P_t) = \{p_{t,0} - m, p_{t,1} - m, ... p_{t,n} - m\}
    \end{gather*}
</pre>